process/block/preprocess/selectionSession.go

```diff
@@ -47,6 +47,11 @@ func NewSelectionSession(args ArgsSelectionSession) (*selectionSession, error) {
 func (session *selectionSession) GetAccountState(address []byte) (state.UserAccountHandler, error) {
 	account, err := session.getExistingAccount(address)
 	if err != nil {
+		// TODO: brainstorm whether we should move the logic around "ErrAccNotFound",
```

Comment:
+1 on keeping as it is, in case other specific handling would be needed in the future